packages/diagrams/frontend/sirius-components-diagrams-reactflow/src/renderer/DiagramRenderer.tsx

```diff
@@ -224,10 +231,10 @@ export const DiagramRenderer = ({ diagramRefreshedEventPayload }: DiagramRendere
         };
         onNodesChange([resetPosition]);
       })}
-      onNodeMouseEnter={(_event: React.MouseEvent<Element, MouseEvent>, node: Node<NodeData>) => {
+      onNodeMouseEnter={useCallback((_event: React.MouseEvent<Element, MouseEvent>, node: Node<NodeData>) => {
```

Comment:
Stop using hook inline like that, make it visible by extracting the function above